.github/workflows/ci.yml

```diff
@@ -17,7 +17,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, ubuntu-20.04]
```

Comment:
`actions/cache@v3` step would need to be updated as well :)

Reply:
ah, no. Sorry! It wasn't using `matrix.os`

Reply:
I am confused, it s creating the matrics for those OS for all those versions correctly. see https://github.com/ExHammer/hammer/pull/79 where we test against both OS all those versions

Reply:
No. What you are seeing that looks correct to you is nothing but a label. Take a look into the logs and you can see how every job is running on top of ubuntu-latest under the hood.

Open the logs for [this job](https://github.com/ExHammer/hammer/actions/runs/7415825187/job/20275684563?pr=79), navigate through "Set up job" > "Operating System" and confirm that it's employing 22.04 despite you would assume it would be 20.04.

Reply:
thanks for reporting this. The fix is not to remove but instead fixing it. I think I see what might be missing

"    runs-on: ${{ matrix.os }}" need to be added